### ISSUE TITLE
Add or change default screen reader keyboard shortcut on MacOS

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#12339]
We have a conflict between turning on RStudio screen reader support and a MacOS VoiceOver screen reader shortcut.
- To turn on RStudio screen reader support, the shortcut is <kbd>Ctrl+Opt+/</kbd>. We now let the user know this shortly after RStudio Desktop starts up.
- The default MacOS VoiceOver shortcut (VO) is <kbd>Ctrl+Opt</kbd>
- <kbd>VO+/</kbd> (by default, <kbd>Ctrl+Opt+/</kbd>) is used for creating a custom label.
- This means that if the user is using the default VO keys, they first need to use <kbd>VO+Tab</kbd> to tell VoiceOver to ignore the next keypress before trying to turn on RStudio screen reader support.
- That is, the user has to press <kbd>Ctrl+Opt+Tab</kbd> and then <kbd>Ctrl+Opt+/</kbd> to turn on RStudio screen reader support.

We should consider coming up with another shortcut for toggling RStudio screen reader support. If possible, we should still let the old shortcut work, as well, for anyone who's used to it with RStudio Server.

**Comment by mikebessuille** [comment]
@gtritchie do you think this is urgent for Elsbeth? If not, we should document this.  We should also update the documentation, regardless, to make sure it says the desktop version supports accessibility.

**Comment by gtritchie** [comment]
I will assign a new shortcut, and make the documentation updates (both the user guide and the support articles).

**Comment by ronblum** [comment]
Verified in RStudio Desktop 2022.12.0-preview+339. 

@gtritchie For RStudio Server, is the shortcut still supposed to be `Alt-Ctrl-/`? I think it should be for backwards compatibility, but I want to make sure this is intentional before closing this.

**Comment by gtritchie** [comment]
@ronblum From the PR:

> QA Notes
> 
> This change applies to both RStudio Desktop and RStudio Server when working on a Mac.

The shortcut has also changed for Server if you are hitting it from a Mac web browser. It is unchanged from Windows or Linux.

**Comment by ronblum** [comment]
It turns out that I didn't update my instance of RStudio Server. Now that I've done so, the new shortcut works there, to.

However, both <kbd>Cmd-Q</kbd> and <kbd>Ctrl-Q</kbd> close RStudio Desktop, so it's too easy to accidentally leave out the <kbd>Shift</kbd> and close the app. So we should come up with a different shortcut.

**Comment by gtritchie** [comment]
Updated to use <kbd>Ctrl+Shift+U</kbd>.

**Comment by ronblum** [comment]
Verified in RStudio Desktop and Server 2022.12.0-preview+341 on MacOS 13.
